test/integration/helpers_test.go

```diff
@@ -123,8 +51,8 @@ func gatewayHealthCheck(ctx context.Context, client *gatewayclient.Clientset, ga
 		case <-tick:
 			tick = nil
 			ch <- func() bool {
-				gw, err := client.GatewayV1().Gateways(namespace).Get(ctx, gatewayName, metav1.GetOptions{})
-				exitOnErr(ctx, err)
+				gw, err := client.GatewayV1beta1().Gateways(namespace).Get(ctx, gatewayName, metav1.GetOptions{})
```

Comment:
Why `beta1`? 🤔

Reply:
Because this PR was waiting for a brave soul to review and so long that Gateway API managed to GA 😆

Reply:
I'll update it